src/main/java/com/github/rodionovsasha/commentservice/exceptions/InactiveUserException.java

```diff
@@ -1,5 +1,9 @@
 package com.github.rodionovsasha.commentservice.exceptions;
 
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.FORBIDDEN)
```

Comment:
This `@ResponseStatus` is a wrong thing here because the exception is located on the services layer which does not know about the presentation details like HTTP